csharp-password-hash/csharp-password-hash/PasswordHashing.cs

```diff
@@ -65,5 +65,33 @@ public string GetHash(string password, HashingConfig hashConfig)
 
             return hashConfig.GenratePerPasswordSalt ? $"{passwordHash}:{salt}" : passwordHash;
         }
+  
+        public HashingConfig GetPossibleConfig(string password, string globalsalt, string saltedPasswordFormat, string inputhash)
+	{
+	     var hashConfig  = new HashingConfig ();
+	     var idx = inputhash.LastIndexOf(':');
```

Comment:
inputhash will be only a hash value. if any salt is used to make this hash then that should be passed into globalsalt param. We can rename the globalsalt param as salt to avoid confusion.

Reply:
Okay. But in that case whether global salt is used or per new salt per password is used that we won't be able to distinguish

Reply:
Please suggest i will do the changes accordingly.

Reply:
We don't need to use GetSaltAndHash method. inputhash should be consider as passwordHash and globalsalt param can be rename as 'salt' and can be used directly into the GetSaltedPassword method.

Reply:
okay i will do the changes mentioned.

Reply:
Please check now.

Reply:
Thanks